site/walkthrough.md

```diff
@@ -0,0 +1,109 @@
+---
+title: weave Walkthrough
+---
+
+## launch DNS by default, IP allocation and naming, both with sensible defaults, rendezvous peering, auto-attach
+
+````bash
+root@a:~# weave launch
+root@a:~# weave run --name=pong -d -ti ubuntu
+````
+
+Weave automatically picked an IP range that isn't used locally (this
+range can also be specified explicitly, as an argument to `weave
```

Comment:
It isn't sufficient to pick a range that isn't used _locally_; it needs to be free on every host you're planning to add to the Weave network.  Which is quite a hard thing for the program to figure out on its own...